java/velox4j/src/main/cpp/main/velox4j/init/Config.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <velox/common/config/Config.h>
+
+namespace velox4j {
```

Comment:
To maintain consistency with other modules such as `facebook::velox::py`, could we change the `velox4j `namespace to `facebook::velox::velox4j`?

Reply:
I'll prefer `facebook::velox4j` since it's not sharing the same project root with Velox C++?

Reply:
@zhztheplayer It appears that most of the Python modules are located in the `velox/python` directory https://github.com/facebookincubator/velox/tree/main/velox/python, with only a few test codes in the `/python `folder https://github.com/facebookincubator/velox/tree/main/python. If we aim to maintain Java at the same level as Python within Velox, should we also organize the functional code similarly by placing it in `velox/java` rather than directly in the `/java` directory?

Reply:
> should we also organize the functional code similarly by placing it in velox/java rather than directly in the /java directory?

Thank you but it's not a good idea from my perspective because it breaks the isolation between Java and C++. My intention was to encapsulate the Java bindings completely within a individual folder, development and maintenance will be made much easier because of doing that.

Reply:
@zhztheplayer It appears that the Java directory in this PR contains both Java and C++ code, rather than just Java code

Reply:
`java` stands for the Java bindings, it's not necessarily for placing Java code only.